### PR TITLE
Workloads page on Insights Advisor in Hybrid Cloud Console

### PR DESCRIPTION
**Pull request by tisnik**
# Description

Workloads page on Insights Advisor in Hybrid Cloud Console

## Type of change

- New test scenario added into existing feature file
- Updated existing test scenario added into existing feature file

## Testing steps

N/A

## Checklist
* [ ] Pylint passes for Python sources
* [ ] sources has been pre-processed by Black
* [ ] updated documentation wherever necessary
* [ ] new tests can be executed both locally and within docker container
* [ ] new tests have been included in scenario list (make update-scenarios)

### CODE REVIEW

File: features/DVO_Recommendations/DVO_recommendations.feature [hunk 1]
```diff
@@ -1,14 +1,22 @@
...
+        Current way of presenting workload related recommendations is not good
+        enough. As a customer using OCP Advisor I'd like to be able to consume
+        Workload related recommendations grouped by namespaces. This means that
+        new UX controls need to be added into Insights Advisor: it would split
+        results of DVO recommendations per namespace and display it to end
+        users.  Notifications should also be sent per namespace (which would
+        enable developers to gain access to results per namespace they own).
 
+        Workload related recommendations should be displayed on page named
+        Workloads.  Entry point to this page is available through the Advisor
```
Comment (epapbak) [inline features/DVO_Recommendations/DVO_recommendations.feature:12]: double white space =)

File: features/DVO_Recommendations/DVO_recommendations.feature [hunk 2]
```diff
@@ -46,7 +54,7 @@ Feature: Current way of presenting workload related recommendations is not good
           | Recommendations     | no                         |
           | Workloads           | yes                        |
      When user select "Workloads" menu item from this sub-menu
-     Then an "Workloads" page should be displayed right of the left menu bar
+     Then a "Workloads" page should be displayed right of the left menu bar
```
Comment (epapbak) [inline features/DVO_Recommendations/DVO_recommendations.feature:57]: "Advisor Workload" page, based on the UX design

Comment (epapbak) [reply features/DVO_Recommendations/DVO_recommendations.feature:57]: comment applies on line 99, 163, and 234 too, if applied

**Review by epapbak — APPROVED** [review]
LGTM

File: features/DVO_Recommendations/DVO_recommendations.feature (hunk 1 above)
Comment (tisnik) [reply features/DVO_Recommendations/DVO_recommendations.feature]: fixed, thank you

File: features/DVO_Recommendations/DVO_recommendations.feature (hunk 2 above)
Comment (tisnik) [reply features/DVO_Recommendations/DVO_recommendations.feature:57]: fixed, thank you

**Review by juandspy — APPROVED** [review]

**Review by joselsegura — APPROVED** [review]

**Pull request merged**